controllers/apply/README.md

```diff
@@ -0,0 +1,29 @@
+# Apply Router
+
+This router handles all the application forms under `/apply`.
+
+There are two key concepts: The form router which is shared across all forms, and form models which are specific to each product.
+
+## dashboard
+
+This directory handles routing and views for the dashboard screens which show people a summary of their applications.
+
+## form-router
+
+This directory contains common router code shared across all forms. It handles common behaviour for eligibility checkers right through to submitting applications to Salesforce but doesn't contain any product specific code.
+
+## awards-for-all
```

Comment:
Does this need a rename now? `under-10k`?

Reply:
Good call